docs-source/examples.rst

```diff
@@ -0,0 +1,250 @@
+.. globus-cli examples master file
+
+Globus CLI Examples
+===================
+
+.. note::
+    The following examples are all written for a Bash shell, if you are running
+    another shell you may need to modify the non globus commands and syntax
+    found in these examples
+
+    The following examples also assume the user has already successfully 
+    authenticated with Globus Auth. To do so run
```

Comment:
Very minor edit, but let's say "logged in to Globus".

`authenticated with Globus Auth` is more correct, but our communications to end-users don't draw distinctions between our services.